components/select_team/components/select_team_item.tsx

```diff
@@ -79,8 +78,8 @@ export default class SelectTeamItem extends React.PureComponent {
                 {this.renderDescriptionTooltip()}
                 <a
                     href='#'
-                    id={Utils.createSafeId(team.display_name)}
-                    onClick={canJoin ? this.handleTeamClick : null}
+                    id={Utils.createSafeId(team.display_name) ?? undefined}
```

Comment:
I haven't seen this pattern before where we coalesce a value to undefined. Do we need this because `createSafeId` can return null, and React complains if we pass a null id? I haven't seen this anywhere before, but I assume it's because we haven't used this in a TS file before.

Maybe we could change the return type of `createSafeId` to return undefined instead of null in the error case to save people from having to do this elsewhere?

Reply:
Thanks. Made this change.